docs/administering/sniproxy.md

```diff
@@ -0,0 +1,146 @@
+# Sharing the HTTPS Port with sniproxy
+
+The purpose of this tutorial is to set up sniproxy so it’s possible to use regular https:// URLs for your main web server, like nginx or Apache, as well as Sandstorm, which has its own HTTPS handling. To make that work, they will need to share port 443.
+
+## Introduction
+
+The purpose of this tutorial is to set up sniproxy so it's possible to use Sandstorm-verified TLS encryption while coexisting with another web server that also uses TLS and share port 443 with it.
```

Comment:
This seems redundant with the paragraph above.

Reply:
Yeah, I removed another paragraph like this. I am not sure how much of this happened when transitioning this from a blog post to a docs page or something. I'll clean it up.